src/poly/commitment/prover.rs

```diff
@@ -20,12 +19,12 @@ impl<C: CurveAffine> Proof<C> {
     /// opening v, and the point x. It's probably also nice for the transcript
     /// to have seen the elliptic curve description and the SRS, if you want to
     /// be rigorous.
-    pub fn create<HBase, HScalar>(
+    pub(crate) fn create<HBase, HScalar>(
```

Comment:
This effectively hides the `Proof` functionality behind the `multiopen` module. I think that's okay, since it provides a strict superset of behavior, but is it what you intended?

Reply:
This was mainly because it would require making `ChallengeZ` public, whereas the `multiopen` prover doesn't take in any challenges at all in its public API. If you're fine with `ChallengeZ` being public, I can make this `pub` again.

Reply:
Fixed.